package.json

```diff
@@ -4,15 +4,20 @@
   "version": "1.1.11-next.1",
   "files": [
     "dist",
-    "src/utils/tailwind",
-    "src/utils/component-classes"
+    "src"
   ],
   "type": "module",
   "style": "dist/fabric.min.css",
   "exports": {
     ".": "./dist/fabric.min.css",
     "./tailwind-config": "./src/utils/tailwind/tailwind.config.js",
-    "./component-classes": "./src/utils/component-classes/index.js"
+    "./component-classes": "./src/utils/component-classes/index.js",
+    "./component-classes/list.js": "./src/utils/component-classes/list.js",
+    "./base.css": "./src/base.css",
+    "./components.css": "./src/components.css",
+    "./utilities.css": "./src/utilities.css",
```

Comment:
These css exports may not be of all that much use since PostCSS doesn't yet support Node package exports so you still need to refer to a resource by its full path. (`@fabric-ds/css/src/components.css` will work, `@fabric-ds/css/components.css` will not). It doesn't hurt though and some tool in future may support package exports so I think this is the correct approach.